product-service/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "product-service",
+  "version": "1.0.0",
+  "description": "",
+  "main": "handler.js",
+  "type": "module",
+  "scripts": {
+    "build": "node esbuild.config.js",
```

Comment:
Again just a nice to have, since you are using yarn, and we will probably have more services: https://classic.yarnpkg.com/lang/en/docs/workspaces/